commands/rm.go

```diff
@@ -100,3 +119,53 @@ func rm(ctx context.Context, dockerCli command.Cli, ng *store.NodeGroup, keepSta
 	}
 	return err
 }
+
+func rmInactive(ctx context.Context, txn *store.Txn, dockerCli command.Cli) error {
+	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
+	defer cancel()
+
+	ll, err := txn.List()
+	if err != nil {
+		return err
+	}
+
+	builders := make([]*nginfo, len(ll))
+	for i, ng := range ll {
+		builders[i] = &nginfo{ng: ng}
+	}
+
+	eg, _ := errgroup.WithContext(ctx)
+	for _, b := range builders {
+		func(b *nginfo) {
+			eg.Go(func() error {
+				if err := loadNodeGroupData(ctx, dockerCli, b); err != nil {
+					return errors.Wrapf(err, "cannot load %s", b.ng.Name)
+				}
+				var kn []store.Node
+				for idx, n := range b.ng.Nodes {
+					d := b.drivers[idx]
+					if d.info == nil || d.info.Status != driver.Running {
```

Comment:
Should be better to check for `driver.Inactive` instead and keep stopped or starting nodes?

Reply:
`driver.Inactive` just seems initial state so unrelated in here. Should check nodes but iiuc you need to check all nodes and rm if none of them are running.

Reply:
I think logic is same for dynamic nodes @morlay

Reply:
> Should check nodes but iiuc you need to check all nodes and rm if none of them are running.

Yes already done: https://github.com/docker/buildx/pull/885/files#diff-aaef9e407115d8cec3e6bde548bd5542bedc65207d4a74e636d68174979a3dc8R157-R164

Reply:
> `driver.Inactive` just seems initial state so unrelated in here.

Yes indeed, just checking node is not in running state seems good.